modules/apps/object/object-test/src/testFunctional/tests/tests/CreateObject.testcase

```diff
@@ -941,12 +941,20 @@ definition {
 	}
 
 	@description = "LPS-135549 - Verify it is possible to view the Details of an Object by clicking on the eye icon"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanViewObjectsDetailsByClickingEyeIcon {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
 
-		// TODO LPS-138813 CanViewObjectsDetailsByClickingEyeIcon pending implementation
+		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+
+		ObjectAdmin.openObjectAdmin();
+
+		CreateObject.viewObject(labelName = "CustomObject");
```

Comment:
Hi @evanilson-santana ,

for this test, you need to view the Object details through the eye (view) option on the kebab menu,

You have to use the `ObjectAdmin.selectCustomObjectFieldKebabMenuOption` macro. If I'm not mistaken, this macro is broken, so you'll have to fix it too.

Thanks!